runtime/src/main/java/com/dylibso/chicory/runtime/HostImportsBuilder.java

```diff
@@ -0,0 +1,94 @@
+package com.dylibso.chicory.runtime;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public final class HostImportsBuilder {
```

Comment:
Nit: we have not "streamlined" yet an approach in this repo:
- external Builder class
- internal `.Builder`

I have a slight preference for the latter but happy with whatever we find an agreement on.
Cc. @electrum @danielperano @bhelx

Reply:
I prefer the internal. Seems cleaner and more common.

Reply:
I changed the commit to reflect the second approach.